projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -239,6 +239,11 @@ export interface FeatureTogglesInterface {
    */
   a11yCloseProductImageBtnFocus?: boolean;
 
+  /**
+   * When enabled use translation fallback keys enabling to use descriptive error message in non-destructive way
+   */
```

Comment:
[MINOR]

Let's explain in details the breaking changes, so customers' developers can quickly understand the impact on their custom codebases. 

```suggestion
  /**
   * In `FormErrorsComponent` it uses more descriptive validation error messages
   * in all UI form fields existing before v2211.25. 
   *
   * 1. The `FormErrorsComponent` uses new i18n keys: 
   * `formErrors.labeled.<validatorName>` instead of `formErrors.<validatorName>`, 
   * for example `formErrors.labeled.required` instead of `formErrors.required`.
   *
   * 2. The existing usages of `CustomFormValidators.passwordValidator` are replaced with
   * an array of new, more specific validators `CustomFormValidators.passwordValidators`
   * (with the plural `...Validators`)
   */
```

Reply:
Changed